Src/Screens/ProfileScreen/Screens/CharityScreen/Viewables/CharityLogo.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CharityLogo.swift
+//  ugglan
+//
+//  Created by Gustaf Gunér on 2019-03-21.
+//
+
+
+import Flow
+import Foundation
+import UIKit
+
+struct CharityLogo {
+    let url: String
```

Comment:
In Swift land a url is never a `String` but a `URL` 😉

Reply:
Swift land never ceases to amaze me 🙈